src/Shouldly.Tests/ShouldContain/EmptyArrayScenario.cs

```diff
@@ -13,7 +13,8 @@ protected override string ChuckedAWobblyErrorMessage
         {
             get
             {
-                return "new int[0] should contain 1 but does not" +
```

Comment:
@JakeGinnivan I've just updated my branch with this. Most tests have a new member called 'target' that is used in both passing and failing scenarios.